src/Particular.ThroughputCollector/BrokerThroughput/BrokerThroughputCollectorHostedService.cs

```diff
@@ -61,20 +61,17 @@ await foreach (var queueName in throughputQuery.GetQueueNames(stoppingToken))
 
         async Task Exec(IQueueName queueName, DateOnly startDate)
         {
-            var endpoint = await dataStore.GetEndpointByName(queueName.QueueName, ThroughputSource.Broker);
+            var endpointId = new EndpointIdentifier();
```

Comment:
This looks wrong, don't we need to set the name and source?

Reply:
Actually, how come we allow to new up a `EndpointIdentifier` without specifying name and source?